### DISCUSSION TITLE
Hacktoberfest topic

### DISCUSSION THREAD

**Discussion by rluvaton** [mobxjs/mobx-state-tree#1804] [General] [answered]
It is that time again. hacktoberfest is totally opt in now:

> A repository/project is considered to be participating in Hacktoberfest if the **`hacktoberfest` topic is present** and is accepting public contributions via pull requests. An individual pull request can also be opted-in directly by adding the `hacktoberfest-accepted` label.

Could you please add the `hacktoberfest` topic?

**Comment by EmilTholin** [comment, marked as answer]
Hi @rluvaton!

We have discussed this amongst the maintainers, and we have decided not to add the `hacktoberfest` topic.
